ls1intum/Artemis
# `Development`: Remove 'tracker' from websocket path

We do not really track something, so the path name is misleading (it was part of the generated Jhipster template)

### Checklist
#### General




### Steps for Testing
1. Log in to Artemis
2. Check that web socket messages are sent normally (e.g. results for programming exercises)
3. Check in the browser console network tab if the web socket messages are sent normally (see screenshot below)
4. Check that logout disconnects the web socket messages
5. Check that login connects web sockets to the server
6. Check that after a network outage (turn off your Internet, wait up to 20s until the disconnect icon appears and then reconnect and wait until the disconnect icon disappears) the web sockets automatically reconnect

<img width="787" alt="image" src="https://user-images.githubusercontent.com/744067/213886942-71fa0c9f-bcc9-4dba-a330-619159eb62a1.png">


### Review Progress






#### Code Review
#### Manual Tests
